### DISCUSSION TITLE
Error #98124 when porting from an existing create-react-app project

### DISCUSSION THREAD

**Discussion by chrsrns** [gatsbyjs/gatsby#32360] [Help] [answered]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Description

The error #98124 is presented when I run `gatsby develop` on an existing `create-react-app` project. Part of the error includes:
>
Generating development JavaScript bundle failed
undefined
If you're trying to use a package make sure that 'undefined' is
installed. If you're trying to use a local file make sure that the
path is correct.

The "undefined" part turns out to be a major roadblock as it doesn't help with the research process of the problem. Any help would be appreciated. The link to the repo is attached.

### Reproduction Link

https://github.com/LifeWatcher123/infinix-website/tree/development/frontend/web-fe

### Steps to Reproduce

1. Clone repository.
2. Install through `npm install` (not sure if necessary when cloning).
3. Run `gatsby develop`

### Expected Result

I'm guessing it should give a link to a hot-reloading development environment accessible from `http://localhost`.

### Actual Result

It gives the error ##98124.

### Environment

```shell
System:
    OS: Linux 5.11 Pop!_OS 21.04
    CPU: (4) x64 AMD A8-7650K Radeon R7, 10 Compute Cores 4C+6G
    Shell: 5.1.4 - /bin/bash
  Binaries:
    Node: 12.21.0 - /usr/bin/node
    npm: 7.5.2 - /usr/bin/npm
  Languages:
    Python: 3.9.5 - /home/flavolite/Eclipse-Workspaces/Web-Workspace/test/infinix-website/env/bin/python
  Browsers:
    Firefox: 89.0.2
  npmPackages:
    gatsby: ^3.9.1 => 3.9.1 
    gatsby-cli: ^3.9.0 => 3.9.0 
  npmGlobalPackages:
    gatsby-cli: 3.9.0
```

**Comment by LekoArts** [comment, marked as answer]
Hi, please see https://www.gatsbyjs.com/docs/porting-from-create-react-app-to-gatsby/

You'll not want to mix CRA and Gatsby in the same project. I'd suggest setting up a new project with Gatsby (`npm init gatsby`) and porting your things over there :)

**Reply by chrsrns** [reply]
I expected as much. I was hoping that I maybe have done something wrong while following [said link](https://www.gatsbyjs.com/docs/porting-from-create-react-app-to-gatsby/). 

Thanks.

**Comment by kylefowler** [comment]
im actually getting the same error when upgrading from gatsby 2 to 3, and its unclear exactly what is going on
